R/Sound_DMR.R

```diff
@@ -625,7 +625,7 @@ run_model <- function(data, i, Output_Frame, formula, model_type,
 #' @return Output_Frame (df) - Output_Frame with the summary statistics from the model
 #' @export
 
-group_DMR <- function(Output_Frame, ZoomFrame_filtered, experimental_design_df, fixed = c('Group'),
+group_DMR <- function(methyl_summary, ZoomFrame_filtered, experimental_design_df, fixed = c('Group'),
```

Comment:
Can you update the doc string for this method to reflect the changed parameter name?

Reply:
Yup, can do..